src/acceptance/api/api_test.go

```diff
@@ -97,6 +97,14 @@ var _ = Describe("AutoScaler Public API", func() {
 			Expect(string(newPolicy)).Should(MatchJSON(policy))
 		})
 
+		It("should succeed to create a valid policy but remove any extra fields", func() {
+			policyWithExtraFields, validPolicy := GenerateDynamicScaleOutPolicyWithExtraFields(1, 2, "memoryused", 30)
+			newPolicy, status := createPolicy(policyWithExtraFields)
```

Comment:
should be be returning a 400 as the schema is incorrect. Or do you think this will break some backward compatiblity ?

Reply:
Yes, I think so. E.g. ages ago a `stats_window_secs` parameter existed and we have users haven't yet removed that parameter from their policies.